src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicApi.cs

```diff
@@ -83,28 +89,58 @@ static MsQuicApi()
         try
         {
             // Check version
-            int arraySize = 4;
+            int arraySize;
+            uint paramSize;
+            int status;
+
+            arraySize = 4;
```

Comment:
It'd be better if these could be consts, e.g.
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAMIR8AB14AbGFADKMgG68wMXAG4aNANr4IAEw7SQDWQGteYgDIQw2SbgCSXXhgC6m2gGYmpBkIYA3jQMIUze3LjYAGYwDLjAAJ4YsQCy5AAUAJSBwaF5kFy4GAy8PAwAglBQ2AmyvABesQC8DCga1Hl58UkwAFQMYhhQDC1F2GCmdpI2cYnJWpXVtQ0wru2docQA7ANDWnRruSEAvkcMZ8ThhdGx3ckMKaRZOR0bJWWLNXWNI63rb3c+rthqMMONJpJpmBZj0FlUvitDq8NttgfskXlTtRjkA==

Reply:
How much does this matter if this is called once per the whole application? Or is this more about setting up good examples/usage patterns?